src/main/java/com/entrevistador/orquestador/dominio/model/enums/FuenteEnum.java

```diff
@@ -0,0 +1,22 @@
+package com.entrevistador.orquestador.dominio.model.enums;
+
+import lombok.Getter;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Getter
+public enum FuenteEnum {
+    ANALIZADOR(new EstadoProcesoEnum[] {EstadoProcesoEnum.AC, EstadoProcesoEnum.AOE,
+            EstadoProcesoEnum.CVA, EstadoProcesoEnum.AE}),
+    GENERADOR_FEEDBACK(new EstadoProcesoEnum[] {EstadoProcesoEnum.GF, EstadoProcesoEnum.GFG}),
+    PREPARADOR_ENTREVISTA(new EstadoProcesoEnum[] {EstadoProcesoEnum.GP, EstadoProcesoEnum.OR,
+            EstadoProcesoEnum.AR, EstadoProcesoEnum.AF});
+
+    private List<EstadoProcesoEnum> estado;
+
+    private FuenteEnum(EstadoProcesoEnum[] estado){
```

Comment:
De acuerdo a Java, el constructor es implícitamente private. 
Una buena practica es evitar la redundancia.
```suggestion
    FuenteEnum(EstadoProcesoEnum[] estado) {
```